src/api/utils.js

```diff
@@ -98,6 +98,15 @@ const addValueAsProp = (data, value, propName) => {
     }
 };
 
+const addAttributeValues = (data, user) => {
+    if (_.isArray(user.attributeValues) && user.attributeValues.length > 0) {
```

Comment:
Any reason to not use Array.isArray() here? I believe it's actually just an alias in lodash.

Reply:
Only reason is consistency. I use a call to `_.isArray` in two other places in this file.

Reply:
OK done